scm/doc/TechGuide/chap_quick.rst

```diff
@@ -4,18 +4,8 @@ Quick Start Guide
 =================
 
 This chapter provides instructions for obtaining and compiling the CCPP
-SCM. The SCM code calls CCPP-compliant physics schemes through the CCPP
-framework code. As such, it requires the CCPP framework code and physics
-code, both of which are included as submodules within the SCM git repository. This
-package can be considered a simple example for an atmospheric model to
-interact with physics through the CCPP.
-
-Alternatively, if one doesn’t have access to or care to set up a machine
-with the appropriate system requirements but has a working Docker
-installation, it is possible to create and use a Docker container with a
-pre-configured computing environment with a pre-compiled model. This is
-also an avenue for running this software with a Windows PC. See section
-:numref:`Section %s <docker>` for more information.
+SCM. We provide instructions on building the code from scratch (:numref:`Section %s <obtaining_code>`), as well as
```

Comment:
Shortened because this is a "quick" start guide? Otherwise, this might be new information from someone unfamiliar with CCPP, etc.

Reply:
Yes that's the idea, I wanted to trim down the less relevant bits in this chapter to make it a "quicker" start. The "Intro" chapter already covers this at a high level, I could expand that intro with this info about submodules specifically if you think that's needed.